Update manage-source-templates.md (#5361)
SumoLogic/sumologic-documentation: docs/send-data/opentelemetry-collector/remote-management/source-templates/manage-source-templates.md

1. Validates if the collector is remotely managed.
1. Validates if the key/value is empty string.
1. Validates the agent version.

> 1. Validates if there is *no* disco tags are added, updated, or deleted.

Reviewer: ```suggestion
1. Validates that *no* disco tags are added, updated, or deleted.
```
